README.md

```diff
@@ -47,24 +49,28 @@ Speaking of contributing, read on!
 
 # Contributing
 
-This project has plenty of interesting and challenging work to do and a wonderful community.
+If you use Athens for your development workflow, we hope that you'll consider contributing back to the project. Athens is widely used and has plenty of interesting work to do, from technical challenges to technical documentation to release management. We have a wonderful community that we would love you to be a part of. [Absolutely everyone is welcome](https://arschles.com/blog/absolutely-everybody-is-welcome/).
+
+The quickest way to get involved is by [filing issues](https://github.com/gomods/athens/issues/new/choose) if you find bugs or find that you need Athens to do something it doesn't.
+
+You also might consider contributing code back to the project. If you're familiar with the GitHub contribution process, you'll probably be familiar with our process. Generally, you may optionally find or submit an issue, and then submit a pull request (PR) to fix that issue. See [here](https://docs.gomods.io/contributing/) for some of the project-specific details.
```

Comment:
The first sentence here is a bit repetitive with the first sentence of the first paragraph. Maybe something like:

> If you'd like to help us tackle the technical challenges, and are familiar with the GitHub contribution process, the Athens community welcomes your contribution!